Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -126,6 +126,11 @@ class ElasticSearchQueryBuilder implements QueryBuilderInterface, ProtectedConte
      */
     protected $result = array();
 
+    /**
+     * @Flow\InjectConfiguration(package="TYPO3.TYPO3CR.Search")
```

Comment:
I agree, will move it. All other ES specific was here: https://github.com/cron-eu/Flowpack.ElasticSearch.ContentRepositoryAdaptor/blob/8e0f19abcd95398c1f08b4624d95fa04c9c079b1/Configuration/Settings.yaml  that's why I put it there too. But it looks strange, that's right.